Azure/azure-cli
# [App Service] `az functionapp create`: Update the default image to `mcr.microsoft.com` for Centauri

**Related command**
`az functionapp create`

**Description**
For Centauri, if the customer does not provide an image, we provide a default value. This default value needs to be updated.

**Testing Guide**
Create a Centauri function app without an image value and then check that the created function app uses the new default image value.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
